www/talp03/sp/Model/ProductDB.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+require_once 'Database.php';
+
+class ProductDB extends Database {
+    
+    public function findAll() {
+        $statement = $this->pdo->prepare('SELECT * FROM products');
+        $statement->execute();
+        return $statement->fetchAll(PDO::FETCH_ASSOC);

```

Comment:
to PDO::FETCH_ASSOC nastavte pomoci atributu pri vytvareni instance pripojeni

Reply:
vyřešeno